### ISSUE TITLE
Issue importing RivGraph in Google Colab

### ISSUE THREAD

**Issue by GustavoWillyNagel** [jonschwenk/RivGraph#72]
Hi. 

I am not being able to import the RivGraph in Google Colab. I tried different ways of doing it, and I failed in all. 

This is my code: https://colab.research.google.com/drive/1FoW42jdMvues7-BL0CyqQrdjQTXHauFd?usp=sharing
 
      # Install Anaconda
      !wget -c https://repo.continuum.io/archive/Anaconda3-5.1.0-Linux-x86_64.sh
      !chmod +x Anaconda3-5.1.0-Linux-x86_64.sh
      !bash ./Anaconda3-5.1.0-Linux-x86_64.sh -b -f -p /usr/local
      !conda install -y --prefix /usr/local -c <<<your wish>>>>
      
      !git clone https://github.com/jonschwenk/RivGraph
      !conda env create --file /content/RivGraph/environment.yml
      !conda activate rivgraph

Is there a better way of doing it?

Thank you in advance.

**Comment by jonschwenk** [comment]
Hi @GustavoWillyNagel! We typically don't provide this kind of support here, but @elbeejay has set up RivGraph in a Colab environment before. See this example: https://colab.research.google.com/drive/18N0Ei07cJ3sM66Fba948c8Ivd8r2sONS. That example is a couple years old so I'm not sure if the dependencies are correct for the latest RivGraph version. @elbeejay feel free to chime in.

**Comment by elbeejay** [comment]
Yeah I agree with @jonschwenk, if that example works then that is fantastic, although the colab kernel and it's properties are really up to Google - the user has limited control over the programming environment. So I think the caveat Jon included is important:

> That example is a couple years old so I'm not sure if the dependencies are correct for the latest RivGraph version.

For a research application I would strongly recommend installing RivGraph locally. It has many (somewhat delicate) dependencies and the last think you'd want is for a change in the underlying Google Colab environment to prevent it from working one day.

**Comment by GustavoWillyNagel** [comment]
Thank you very much. 

It worked. 

Best wishes.
